tcms/testruns/templates/testruns/get.html

```diff
@@ -78,26 +71,33 @@ <h2 class="card-pf-title" style="text-align: left">
 
                 <h2 class="card-pf-title" style="text-align: left">
                     <span class="fa fa-calendar"></span>{% trans 'Started at' %}:
-                    {{ object.start_date }}
-                </h2>
+                    {% if object.start_date == None %}
+                        <button class="btn btn-success btn-xs" type="button" id="start-button"
+                                data-toggle="tooltip" data-placement="top" title="{% trans 'Start' %}">
+                            <i class="fa fa-check" style="color: white; margin: auto; "></i>
+                        </button>
+                    {% endif %}
+                        <span class="start-date">
+                            {{ object.start_date|default:'-' }}
+                        </span>
 
+                </h2>
                 <h2 class="card-pf-title" style="text-align: left">
                     <span class="fa fa fa-calendar-o"></span>{% trans 'Planned stop' %}:
                     {{ object.planned_stop|default:'-' }}
                 </h2>
-
                 <h2 class="card-pf-title" style="text-align: left">
                     <span class="fa fa-calendar-check-o"></span>{% trans 'Finished at' %}:
+                    {% if object.stop_date == None %}
+                        <button class="btn btn-warning btn-xs" type="button" id="stop-button"
```

Comment:
I tried moving this button to the right with `float:right` and it looks a bit better but then the card height gets shorter and the button  goes outside of the card. If you can figure out how to make it look presentable I'd prefer the buttons to the right. If not leave it as is.